src/components/TreemapText.tsx

```diff
@@ -0,0 +1,132 @@
+import { max } from 'd3-array';
+import React from 'react';
+
+import { stringWidth } from '../utils';
+
+interface TreemapTextProps {
+  datum: any;
+  dropShadowFill?: string;
+  fill?: string;
+  filterDefsUrl?: string;
+  fontSizeExtent?: [number, number];
+  fontMargin?: number;
+  fontPadding?: number;
+  fontSize?: number;
+  label: string | number;
```

Comment:
Not that this is necessarily something that we want to do now. But I was thinking that it might be nice to allow for no input of label so that if someone wanted they could choose to display label or now depending on including or excluding label in the fieldAssessor. Thoughts?

Reply:
Sure! What about if the `label` is just figured out by its parent and put into it's `{children}`:
```
<TreemapText
  {...allTheCorrectProps}
>
  {props.children} // <- then they could decide, quite declaratively to put them here or not.
</ TreemapText>
```
Would that work? I think that'd leave it nice and open for the user (I believe the `children` props is optional by default from the React.Component class). It would probably also be more intuitive to use.